en/config/safety.md

```diff
@@ -0,0 +1,225 @@
+
+
+## QGroundControl Safety Setup {#qgc_safety_setup}
+
+The *QGroundControl* Safety Setup page is accessed by clicking the *QGroundControl* **Gear** icon (Vehicle Setup - top toolbar) and then **Safety** in the sidebar). This includes the most important failsafe settings (battery, RC loss etc.) and the settings for the return actions *Return* and *Land*.
+
+![Safety Setup (QGC)](../../images/qgc/setup/safety_setup.png)
+
+
+### Low Battery Failsafe
+
+The low battery failsafe is triggered when the battery capacity drops below one (or more warning) level values. 
+
+![Safety - Battery (QGC)](../../images/qgc/setup/safety_battery.png)
+
+The most common configuration is to set the values and action as above (with `Warn > Failsafe > Emergency`).
+With this configuration the failsafe will trigger warning, then return, and finally landing if capacity drops below the respective levels.
+ 
+It is also possible to set the *Failsafe Action* to warn, return, or land. 
+In this case the vehicle will perform the specified action when the *Warn* failsafe level is reached.
+
+The settings and underlying parameters are shown below.
+  
+Setting | Parameter | Description
+--- | --- | ---
+Failsafe Action | [COM_LOW_BAT_ACT](../advanced_config/parameter_reference.md#COM_LOW_BAT_ACT) | Warn, Return, or Land based when capacity drops below Battery Warn Level, OR Warn, then return, then land based on each of the level settings below. 
+Battery Warn Level | [BAT_LOW_THR](../advanced_config/parameter_reference.md#BAT_LOW_THR) | Percentage capacity for warnings (or other actions).
+Battery Failsafe Level | [BAT_CRIT_THR](https://docs.px4.io/en/advanced_config/parameter_reference.html#BAT_CRIT_THR) | Percentage capacity for Return action.
+Battery Emergency Level | [BAT_EMERGEN_THR](../advanced_config/parameter_reference.md#BAT_EMERGEN_THR) | Percentage capacity for triggering Land (immediately) action.
+
+
+### RC Loss Failsafe
+
+The RC Loss failsafe is triggered if the RC transmitter link is lost.
+
+![Safety - RC Loss (QGC)](../../images/qgc/setup/safety_rc_loss.png)
+
+The settings and underlying parameters are shown below.
+
+Setting | Parameter | Description
+--- | --- | ---
+RC Loss Timeout | [COM_RC_LOSS_T](../advanced_config/parameter_reference.md#COM_RC_LOSS_T) | Amount of time after losing the RC connection before the failsafe will trigger.
+Failsafe Action | [NAV_RCL_ACT](../advanced_config/parameter_reference.md#NAV_RCL_ACT) | Disabled, Loiter, Return, Land, RC Auto Recovery, Terminate, Lockdown.
+RC Loss Loiter Time | [NAV_RCL_LT](../advanced_config/parameter_reference.md#NAV_RCL_LT) | If the *Failsafe Action* (`NAV_RCL_ACT`) is set to *CASA Outback Challenge rules* this sets the loiter time after RC loss.
```

Comment:
FYI, there is no way to set this in QGC (ie if you set the action to Casa Rules it does not appear). Worth raising a bug report over?

Reply:
Yes I think so.

Reply:
Thanks. Done https://github.com/mavlink/qgroundcontrol/issues/6428